### ISSUE TITLE
deno doc giving incorrect class name on classes that have a default export

### ISSUE THREAD

**Issue by Eyoatam** [denoland/deno_doc#87]
when I tried to generate a doc for class that has a default export it gave the class a name of default

Example:
```ts
export default class greeter {
  greet(name: string) {
    console.log(`Hello ${name}`);
  }
}

```

**Comment by lucacasonato** [comment]
This is correct. From the users perspective they are importing default right?

**Comment by Eyoatam** [comment]
> This is correct. From the users perspective they are importing default right?

yes they will be importing greet like 
```ts
import greeter from "..."
```
I think it would be good if it keeps the name of the and not change it to `default`

**Comment by jaydenseric** [comment]
The name of a member vs what it's exported as are two different things, the name of a class doesn't change between being a named export vs a default export. For packages that provide multiple deep import paths to default export classes, we need a way for their original class names to be visible in documentation.

**Comment by jaydenseric** [comment]
Somewhat related: https://github.com/denoland/deno_doc/issues/116 .

**Comment by crowlKats** [comment]
Closing as this was fixed via #510
